OWNERS_ALIASES

```diff
@@ -1,31 +1,44 @@
 aliases:
-  toc:
+  technical-oversight-committee:
   - evankanderson
   - grantr
   - markusthoemmes
   - mattmoor
-  - tcnghia
+  - rhuss
```

Comment:
while updating, consider removing mattmoor?

Reply:
There will be a PR tomorrow that will take Matt out (the one that you approved in https://github.com/knative/community/pull/552)

Removed now.